### PR TITLE
Add Lnk.Bio

### PR DESCRIPTION
**Pull request by andreaolivato**
Hi,

This is a contribution request for [Lnk.Bio](https://lnk.bio)

JSON has been formatted according to standard and the SVG logo compressed via SVGOMG.

Lnk.Bio global rank on SimilarWeb is #23,712

### CODE REVIEW

File: img/l/lnk.bio.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...-background:new 0 0 512 512" version="1.1" viewBox="0 0 512 512"><style>.st0{display:none;fill:#b996f7}</style><path d="M419.2 512H92.8C41.7 512 0 470.3 0 419.2V92.8C0 41.7 41.7 0 92.8 0h326.5c51 0 92.8 41.7 92.8 92.8v326.5c-.1 51-41.8 92.7-92.9 92.7z" style="fill:#b996f7"/><path d="m138.4 254.1 20.6-20.6c5.5-5.5 14.9-1.9 15.2 5.9.4 10 2.1 19.8 5.4 29.3 1.1 3.2.3 6.8-2.1 9.2l-7.3 7.3c-15.6 15.6-16.1 41-.6 56.7 15.5 15.8 40.9 16 56.7.5l.2-.2 37.3-37.4c15.6-15.6 15.6-41 0-56.6-1.8-1.8-3.7-3.4-5.8-4.8a8.74 8.74 0 0 1-3.9-7c-.2-6.2 2.1-12.2 6.5-16.6l11.7-11.7c3.1-3.1 7.9-3.5 11.4-1 38.3 26.7 47.6 79.5 20.9 117.7-2.8 4-5.9 7.7-9.3 11.1l-.2.2-37.3 37.3c-33 33-86.6 33-119.6 0s-32.8-86.3.2-119.3z" style="opacity:.4;fill:#fff;enable-background:new"/><path d="M341.7 226.8c15.6-15.6 16.1-41 .6-56.7-15.5-15.8-40.9-16-56.7-.5l-.2.2-37.3 37.3c-15.6 15.6-15.6 41 0 56.6 1.8 1.8 3.7 3.4 5.8 4.8 2.4 1.6 3.8 4.2 3.9 7 .2 6.2-2.1 12.2-6.5 16.6l-11.8 11.7c-3.1 3.1-7.9 3.5-11.4 1-38.3-26.7-47.7-79.5-20.9-117.7 2.8-4 5.9-7.7 9.4-11.1l.2-.2 37.3-37.3c33-33 86.6-33 119.6 0s33 86.6 0 119.6L353 278.6c-5.5 5.5-14.9 1.9-15.2-5.9-.4-10-2.1-19.8-5.4-29.3-1.1-3.2-.3-6.8 2.1-9.2l7.2-7.4z" style="fill:#fff"/></svg>
```
Comment (Carlgo11) [inline img/l/lnk.bio.svg]: Reran the image through [SVGOMG](https://jakearchibald.github.io/svgomg/).
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><rect width="512" height="512" fill="#b996f7" ry="88.48"/><path fill="#ffffff66" d="m138.4 254.1 20.6-20.6c5.5-5.5 14.9-1.9 15.2 5.9.4 10 2.1 19.8 5.4 29.3 1.1 3.2.3 6.8-2.1 9.2l-7.3 7.3c-15.6 15.6-16.1 41-.6 56.7a40.1 40.1 0 0 0 56.7.5l37.5-37.6a40.08 40.08 0 0 0-5.8-61.4 8.74 8.74 0 0 1-3.9-7c-.2-6.2 2.1-12.2 6.5-16.6l11.7-11.7c3.1-3.1 7.9-3.5 11.4-1 38.3 26.7 47.6 79.5 20.9 117.7-2.8 4-5.9 7.7-7.7 9.5l-39.1 39.1c-33 33-86.6 33-119.6 0s-32.8-86.3.2-119.3z"/><path fill="#fff" d="M341.7 226.8c15.6-15.6 16.1-41 .6-56.7a40.1 40.1 0 0 0-56.7-.5l-.2.2-37.3 37.3a40.08 40.08 0 0 0 5.8 61.4c2.4 1.6 3.8 4.2 3.9 7 .2 6.2-2.1 12.2-6.5 16.6l-11.8 11.7a8.83 8.83 0 0 1-11.4 1c-38.3-26.7-47.7-79.5-20.9-117.7 2.8-4 5.9-7.7 9.4-11.1l.2-.2 37.3-37.3c33-33 86.6-33 119.6 0s33 86.6 0 119.6L353 278.6c-5.5 5.5-14.9 1.9-15.2-5.9-.4-10-2.1-19.8-5.4-29.3-1.1-3.2-.3-6.8 2.1-9.2l7.2-7.4z"/></svg>
```

File: entries/l/lnk.bio.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Lnk.Bio": {
+    "domain": "lnk.bio",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.lnk.bio/en/articles/8372705-how-to-add-2fa-security",
```
Comment (Carlgo11) [inline entries/l/lnk.bio.json]: Only this part of the URL is needed :wink: 
Have any recovery instructions as well that could be linked to using the `recovery` element?
```suggestion
    "documentation": "https://help.lnk.bio/en/articles/8372705",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by andreaolivato** [comment]
Thanks for the suggestions, I've approved them and added the `recovery` documentation in the same shortened URL format.

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes it takes longer due to server and browser cache.

**Pull request merged**